buildspecs/al-maintenance-release-config.yml

```diff
@@ -0,0 +1,47 @@
+version: 0.2
+
+phases:
```

Comment:
Have we tested this script?

Reply:
Yep, I was able to build a CodeBuild project along with this buildspec file within my personal dev account. Happy to showcase it offline!

Reply:
Here's the corresponding working branch in my forked repo that's testing these changes. Currently the CodeBuild project will be kicked off when this branch is merged into another branch called `AmazonLiuxLegacyTest`. When these changes are merged, it'll be the `AmazonLinuxLegacy` branch instead and we'll create a "dev" branch (e.g. al1-patch).

https://github.com/mye956/amazon-ecs-agent/tree/al1-patch-dummy